src/styles/utils/variables.scss

```diff
@@ -0,0 +1,11 @@
+$start-day: ('mon' 0, 'tue' 1, 'wed' 2, 'thu' 3, 'fri' 4, 'sat' 5, 'sun' 6);
+$cellSize: 100px;
+$cellCollor: #eee;
+$hoverCollor: #ffbfcb;
```

Comment:
Similarly, there is a typo in the variable name `$hoverCollor`. It should be corrected to `$hoverColor`.